src/log/log_slot.c

```diff
@@ -545,6 +544,8 @@ __wt_log_slot_join(WT_SESSION_IMPL *session, uint64_t mysize, uint32_t flags, WT
         unbuffered = true;
         F_SET(myslot, WT_MYSLOT_UNBUFFERED);
     }
+
+    time_start = 0;
```

Comment:
Why move initialization to here?

Reply:
It's not needed before then, that is, initialize close to the first use of the variable?

Reply:
I always thought our style was to initialize next to where we declare.

Reply:
Yes and no? I mean, `for (i = 0;` isn’t initialized next to where we declare? I mean, you’re not wrong — I’ll swap it back.

Reply:
Fixed in 5e2e2a6e4.